### ISSUE TITLE
Pipeline init errors when accepting pipeline name suggestion generated from branch or repo name with uppercase characters

### ISSUE THREAD

**Issue by matthewhembree** [aws/copilot-cli#3435]
When running `copilot pipeline init` and the branch or repo name have uppercase characters, the init will fail when accepting the suggestion.

Reproduction/Output:
```
$ copilot version
version: v1.16.0, built for darwin
$ git clone git@github.com:aws-samples/aws-copilot-sample-service.git demo-app
$ cd demo-app
$ git checkout -b MAIN-2
$ copilot init --app demo                \
  --name api                             \
  --type 'Load Balanced Web Service'     \
  --dockerfile './Dockerfile'
$ copilot pipeline init
Only one git remote detected. Your pipeline will follow 'git@github.com:aws-samples/aws-copilot-sample-service'.
Your pipeline will follow branch 'MAIN-2'.

  What would you like to name this pipeline? [? for help] (aws-copilot-sample-service-main-2)
```

After pressing enter:
```
...
$ copilot pipeline init
Only one git remote detected. Your pipeline will follow 'git@github.com:aws-samples/aws-copilot-sample-service'.
Your pipeline will follow branch 'MAIN-2'.
X Sorry, your reply was invalid: pipeline name aws-copilot-sample-service-main-2 is invalid: value must start with a letter, contain only lower-case letters, numbers, and hyphens, and have no consecutive or trailing hyphen

  What would you like to name this pipeline? [? for help] (aws-copilot-sample-service-main-2)

```

Pressing enter again results in the CLI being unresponsive. Typing input will result in the CLI continuing. 

Some background about my use case:
Specifically: My org uses Jira automation based off the branch name; which is the Jira ID. (e.g. `ABC-123`) This only came up during our adoption of Copilot where our pipeline deployment branch isn't main, but a feature branch following the Jira ID convention. It's probably uncommon for anyone to use branch names other than: `main`, `master`, `develop`; but I am curious if there is a broader issue with the validator for other suggested values. Maybe this issue might be more significant now that multiple pipelines are supported (v1.16.0) and users get more creative with that feature.

.
.
Thanks!

**Comment by dannyrandall** [comment]
Hey @matthewhembree, thanks for catching this and reporting it!

After a quick glance, it looks like this bug is caused by us formatting the suggestion with the faint color. We will have a fix in the next release.

In the meantime, you can type the suggested name (or a different name) and it should work.

**Comment by huanjani** [comment]
The fix is now released in v1.17.0: https://github.com/aws/copilot-cli/releases/tag/v1.17.0.
Thanks for bringing this bug to our attention!
